src/mumble/main.cpp

```diff
@@ -389,6 +394,23 @@ int main(int argc, char **argv) {
 			} else if (args.at(i) == QLatin1String("-c") || args.at(i) == QLatin1String("--config")) {
 				//	We already parsed these arguments above, so just skip over them here
 				++i;
+			} else if (args.at(i) == QLatin1String("--default-certificate-dir")) {
+				if (i + 1 < args.count()) {
+					qdCert = QDir(args.at(i + 1));
+					// I suppose we should really be checking whether the directory is writable here too,
+					// but there are some subtleties with doing that:
+					// (https://doc.qt.io/archives/qt-4.8/qfile.html#platform-specific-issues)
```

Comment:
I would refer to the updated documentation: https://doc.qt.io/qt-5/qfile.html#platform-specific-issues